### ISSUE TITLE
[bug] Can't use responsive space

### ISSUE THREAD

**Issue by lobor** [tamagui/tamagui#1270]
### Current Behavior

This always uses $4:
```
<View 
  space="$4"
  $md={{
    space: '$5',
  }}
/>
```

Related to https://github.com/tamagui/tamagui/issues/304

### Tamagui Version

```markdown
1.31.2
```

### Reproduction

```markdown
on sandbox
```

**Comment by natew** [comment]
Will be fixed in c251e6b3c0f4522c38ed8d0cf4a2ae2a6f9b2cac

**Comment by lobor** [comment]
It's resolve, thanks
